tests/test_cmdline_options.py

```diff
@@ -60,7 +60,7 @@
 """
 )
 venvs = [str(project_with_no_issues / ".venv")]
-other = ["--generate-toml-file", "--version"]
+other = ["--generate-toml-config", "--version", "--install-deps"]
```

Comment:
I'm struggling to see what (if anything) this is used for? Can we remove the entire line?

Reply:
This is not used now anywhere, it is a reminder for our future selves to add `other` to the tested CLI options.